### PR TITLE
Fix use cache example line highlights

### PR DESCRIPTION
**Pull request by gaearon**
Have not tested

### CODE REVIEW

File: docs/02-app/02-api-reference/01-directives/use-cache.mdx [hunk 1]
```diff
@@ -311,7 +311,7 @@ This is recommended for applications that previously used the [`export const dyn
 
 ```tsx filename="app/layout.tsx"
 "use cache"
-import { unstable_cacheLife as cacheLife} from 'next/cache'
+import { unstable_cacheLife as cacheLife } from 'next/cache'
```
Comment (gaearon) [inline docs/02-app/02-api-reference/01-directives/use-cache.mdx:314]: driveby style fix

File: docs/02-app/02-api-reference/01-directives/use-cache.mdx [hunk 2]
```diff
@@ -268,7 +268,7 @@ A `cacheTag` is used in combination with [`revalidateTag`](/docs/app/api-referen
 
 In the below example the `getData` function uses the “weeks” cache profile, and defines a `cacheTag` on the functions cached output:
 
-```tsx filename="app/actions.ts" highlight={4,5}
+```tsx filename="app/actions.ts" highlight={8,9}
```
Comment (gaearon) [inline docs/02-app/02-api-reference/01-directives/use-cache.mdx:271]: fix

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 1015c085c9f5363e9308bb91973deb1d1ef65a43

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by delbaoliveira — APPROVED** [review]
Thank you!
